container-onnxruntime-bundle/src/main/java/ai/onnxruntime/vespa/OnnxBundleActivator.java

```diff
@@ -0,0 +1,51 @@
+// Copyright Yahoo. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.                                                                                                                           
+package ai.onnxruntime.vespa;
```

Comment:
Why not `vespa.ai.onnxruntime`?

Reply:
will put the bundle activator in "our" package namespace (ai.vespa.*)